ui/packages/catalog-ui-search/src/main/webapp/react-component/container/sharing/sharing.tsx

```diff
@@ -255,6 +260,18 @@ export class Sharing extends React.Component<Props, State> {
     this.componentDidMount()
   }
 
+  unSubscribeUsers = () => {
+    this.usersToUnsubscribe.forEach(async user => {
+      await fetch(
+        `/search/catalog/internal/unsubscribe/` + this.props.id + '/' + user,
```

Comment:
```suggestion
        `./internal/unsubscribe/` + this.props.id + '/' + user,
```
I don't think this will pass the linter, in ddf you have to use relative urls.